### PR TITLE
add Slider component

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add code for the Slider component in the development branch.

This is how it looks,
- without focus, 
image

- with focus, 
image
 
Please review this PR, and give your feedback.

### CODE REVIEW

**Review by keshavkumar-in — APPROVED** [review]
Everything looks good to me, good to merge.

File: css/components/Slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+.slider:focus {
+  outline: none;
+}
+
+/** Chromium based **/
+.slider::-webkit-slider-runnable-track {
+  background: var(--COLOR-GLOSSY-BLUE-100);
+  border-radius: 0.5rem;
+  height: 0.5rem;
```
Comment (ShekharDhangar) [inline css/components/Slider/slider.css:17]: Can use px units instead of rem for less than 1rem

File: css/components/Slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  border: thin solid var(--COLOR-GLOSSY-BLUE-700);
+  outline: thin solid var(--COLOR-GLOSSY-BLUE-700);
+  outline-offset: 0.25rem;
+}
+
+/** for Firefox **/
+.slider::-moz-range-track {
+  background: var(--COLOR-GLOSSY-BLUE-100);
+  border-radius: 0.5rem;
+  height: 0.5rem;
```
Comment (ShekharDhangar) [inline css/components/Slider/slider.css:44]: Same can use px instead of rem

**Review by 05him — APPROVED** [review]
Code and UI everything looks awesome to  me , Its Good To Go

**Review by ShekharDhangar — APPROVED** [review]
Apart from the suggested changes, looks good

File: css/components/Slider/slider.css (hunk 1 above)
Comment (hsnice16) [reply css/components/Slider/slider.css:17]: Thanks for pointing this out, Shekhar. actually, I am using rems everywhere that's why I am maintaining that consistency. And Vikram also said we can use px for less than 10px values, but it is not necessary. that's why I am using rems.

**Comment by hsnice16** [comment]
Thank you for reviewing this PR.

**Pull request merged**